WhaleSpotting/Controllers/WhaleSightingController.cs

```diff
@@ -27,4 +28,18 @@ public IActionResult GetById([FromRoute] int Id)
             return NotFound();
         }
     }
+
+    [HttpGet("allpending")]
+    public IActionResult GetPendingSightings()
+    {
+        List<WhaleSighting> whaleSightingsPendingList = _whaleSightingService.GetPendingSightings();
+        List<WhaleSightingResponse> whaleSightingsPendingListResponses = new List<WhaleSightingResponse>();
+
+        foreach (WhaleSighting ws in whaleSightingsPendingList)
+        {
+            var ourWsResponse = new WhaleSightingResponse(ws);
+            whaleSightingsPendingListResponses.Add(ourWsResponse);
+        }
+        return Ok(whaleSightingsPendingListResponses);
```

Comment:
Can we handle potential errors in the controller? Eg. if no pending sightings exist
```suggestion
        try
        {
             return  _whaleSightingService.GetPendingSightings();
        }
        catch (ArgumentOutOfRangeException)
        {
            return NotFound();
        }
```